pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "uvdat-flood-sim"
+description = "An end-to-end dynamic flood simulation."
+readme = "README.md"
+requires-python = ">=3.10"
+license = "Apache-2.0"
+license-files = ["LICENSE", "NOTICE"]
+maintainers = [{ name = "Kitware, Inc.", email = "kitware@kitware.com" }]
```

Comment:
Let me know if this is appropriate. To use this software at all, however, we need to declare a license.

Reply:
@annehaley will verify license questions.